ci/build_container/build_container.sh

```diff
@@ -5,13 +5,7 @@ set -e
 # Setup basic requirements and install them.
 apt-get update
 apt-get install -y wget software-properties-common make cmake git python python-pip \
-  clang-format-3.6 bc libtool automake zip time
-apt-get install -y golang
-# For debugging.
-apt-get install -y gdb strace
-add-apt-repository -y ppa:ubuntu-toolchain-r/test
-apt-get update
-apt-get install -y g++-4.9
+  clang-format-3.6 bc libtool automake zip time golang g++ gdb strace
 # clang head (currently 5.0)
 wget -O - http://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
```

Comment:
we could save some time by doing all the repo adding and coalescing the apt-get update